core-plugin/src/com/google/gct/idea/debugger/CloudDebugRunConfiguration.java

```diff
@@ -58,7 +62,12 @@ protected CloudDebugRunConfiguration(Project project, @NotNull ConfigurationFact
 
   @Override
   public final RunConfiguration clone() {
-    final CloudDebugRunConfiguration configuration = (CloudDebugRunConfiguration)super.clone();
+    if (this == RunManager.getInstance(getProject()).getConfigurationTemplate(this.getFactory()).getConfiguration()) {
```

Comment:
It uses the same clone method to create instances of existing run configurations for editing, in the run configuration editor.